.drone.yml

```diff
@@ -87,7 +87,7 @@ pipeline:
       - docker-compose -f docker-compose-build.yml build --no-cache nodejs python-dev python-prod
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock
-      - /tmp/cache/drone/stroyprombeton/front_build/${DRONE_COMMIT_SHA}:/usr/app/front/build
+      - /tmp/cache/drone/stroyprombeton/front_build/${DRONE_COMMIT_SHA}:/drone/src/github.com/fidals/stroyprombeton/front/build
```

Comment:
it's not clear why we are needed this mapping.
`/drone/src/github.com/fidals/stroyprombeton/front/build` should be visible from every pipeline container according to doc http://docs.drone.io/workspace/

Reply:
@duker33 We alreade have an [issue](https://github.com/fidals/refarm-site/issues/132) to resolve it

Reply:
@artemiy312 , but you reused this code here :)

Ok, let's discuss it in task https://github.com/fidals/refarm-site/issues/132

Reply:
@duker33 
>The base attribute defines a shared base volume available to all pipeline steps. This ensures your source code, dependencies and compiled binaries are persisted and shared between steps.
 
I do not know default value of `base` attribute